bridge_adaptivity/module/tests/test_grading_policy.py

```diff
@@ -82,14 +82,18 @@ def test_grading_policy_has_name(self, test_cls, public_name):
         self.assertEqual(getattr(test_cls, 'public_name'), public_name)
 
     @mock.patch('module.policies.base.BaseGradingPolicy._get_points_earned_trials_count')
+    @mock.patch('module.policies.policy_points_earned.PointsEarnedGradingPolicy._get_points_earned_trials_count')
```

Comment:
Please move `patch` decorators directly under the test function for explicit usage of the function's parameters